blog/posts/basics-performance.md

```diff
@@ -0,0 +1,113 @@
+
+<!-- more -->
+
+## Why performance matters
+
+We must keep in mind that it is the end-user's own perception that ultimately determines what we call _user experience_. The time it takes for a service to become usable, as well as its general responsiveness, bear a lot of weight on this perception. Helpful features, great design and other prominent characteristics all become irrelevant when an online service is so slow that users navigate away. 
+
+You can build the best web application in the world, but be mindful that each user will have a specific amount of time they are willing to invest in your service to solve their problems. Exceed that amount, and you risk losing them to a different, more performant solution. This is even truer for new users, who haven't yet been given proof of the quality of your service, and are essentially investing their time up-front, hoping for a return.
+
+### A competitive differentiator
+
+There is a brighter side to the topic: if low performance can sink an online platform, high performance can very well help it rise to the top. Speed and responsiveness can be a differentiating characteristic for a service, prompting users to choose it over the competition. Therefore an investment in this area will almost always pay off. Some notorious real-world examples from known businesses include:
+
+1. Pinterest decreasing wait time for their users, [increasing both traffic and conversions](https://medium.com/@Pinterest_Engineering/driving-user-growth-with-performance-improvements-cfc50dafadd7).
+2. Zalando applying small improvements in load time and finding a direct correlation with [increased revenue per session](https://engineering.zalando.com/posts/2018/06/loading-time-matters.html).
+3. The BBC discovering that every extra second that a page took to load led to 10% of [users leaving the page](https://www.creativebloq.com/features/how-the-bbc-builds-websites-that-scale).
+
+## Measuring performance
+
+Given the importance of page performance, it is no coincidence that browsers expose a ton of insights into [performance metrics](https://web.dev/metrics/). Being aware of how your application scores against these _across time_ will provide you the feedback you need to keep it performant for your users. There are several approaches that can be combined to achieve the best results:
+
+1. _Real user monitoring_ to understand what performance actual end-users of your service are experiencing.
+2. _Synthetic monitoring_ to proactively gather intel on service performance, as well as to find issues before users stumble into them.
+3. _Performance testing_ to avoid releasing performance regression to production in the first place.
+4. _Regular audits_ to get an overview of your page's performance and suggestions on how to improve it, e.g. with tools such as [Google Lighthouse](https://developers.google.com/web/tools/lighthouse).
+
+## Performance with headless tools
```

Comment:
I'd want to get to "the meat" of the article a little sooner. The first three paragraphs are useful and good, but maybe  just one or two sentences too long? Don't really know how to cut it down.